_posts/2021-09-02-gsoc-final-report-orion.md

```diff
@@ -0,0 +1,194 @@
+
+### In pursuit of user number one
+PR [#36]
+
+PR #36 introduced a [tutorial] to walk users the basic functionality of solvation_analysis and how to respond to common errors. If you are interested in using the `solvation_analysis`, this is definitely the best place to start!
+
+### Release and versioning
+PR [#37]
+
+I was shocked by how easy it is to release a package to PyPI. For this PR, I learned to use [versioneer] and [twine] for Python release management. With just a few terminal commands and file edits, I was able to set up a functioning release management system and put out version 0.1.1.
+
+
+## Solvation Analysis in Practice
+
+Since solvation analysis is up on PyPI, it can be installed with a pip!
+
+```
+>>> pip install solvation_analysis
+```
+
+As in any MDAnalysis workflow, we start by importing the package and instantiating an MDAnalysis Universe.
+
+```python
+# imports
+import MDAnalysis as mda
+
+# define paths to data
+data = "../solvation_analysis/tests/data/bn_fec.data"
+traj = "../solvation_analysis/tests/data/bn_fec_short_unwrap.dcd"
+
+# instantiate Universe
+u = mda.Universe(data, traj)
+```
+
+Next, we need to define the solute and solvents of interest. The example system is a LiPF<sub>6</sub> electrolyte with BN and FEC as solvents. Here, we treat the Li<sup>+</sup> ion as the solute and all other molecules as solvents.
+
+```python
+# define solute AtomGroup
+li_atoms = u.atoms.select_atoms("type 22")
+
+# define solvent AtomGroups
+PF6 = u.atoms.select_atoms("byres type 21")
+BN = u.atoms.select_atoms("byres type 5")
+FEC = u.atoms.select_atoms("byres type 19")
+```
+
+Once we have the universe, solute, and solvents defined, it is quite simple to instantiate and run a solution! Solution, like all MDA analysis classes, subclasses the [AnalysisBase] class and requires `Solution.run()` call to initiate the analysis.
+
+```python
+# instantiate solution
+from solvation_analysis.solution import Solution
+
+solution = Solution(li_atoms, {'PF6': PF6, 'BN': BN, 'FEC': FEC})
+
+solution.run()
+```
+
+Over the course of a few seconds, the solvation radii and the solvation shell of each ion are automatically identified and stored in a `pandas.DataFrame` (as described above). We can then examine how well the solvation radii were characterized by calling `solution.plot_solvation_radius('<<solvent name>>')`. For example,
+
+```python
+# we just need this to display our plot
+import matplotlib.pyplot as plt
+
+solution.plot_solvation_radius('BN')
+plt.show()
+```
+![bn plot]({{site.images}}/final_report_orion/bn_plot.png)
+
+
+Now that we have a completed solution, we can easily print the ion pairing, coordination numbers, and solvation radii.
+
+```python
+>>> solution.coordination.cn_dict
+{'BN': 4.33, 'FEC': 0.25, 'PF6': 0.12}
+>>> solution.pairing.pairing_dict
+{'BN': 1.0, 'FEC': 0.21, 'PF6': 0.12}
+>>> solution.radii
+{'PF6': 2.60, 'BN': 2.61, 'FEC': 2.43}
+```
+
```

Comment:
Would be great if you could do just a sentence or two describing the science here. eg:
"We can see from this data that ....., coordination numbers, .... something something pairing...... something something radii."  :)

Reply:
Good point. Added a couple sentences!